src/GraphQL.Tests/Types/Collections/SchemaTypesTests.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using GraphQL.StarWars;
+using GraphQL.StarWars.IoC;
+using GraphQL.StarWars.Types;
+using Moq;
+using Xunit;
+
+namespace GraphQL.Tests.Types.Collections
+{
+    public class SchemaTypesTests
+    {
+        [Fact]
+        public void does_not_request_instance_more_than_once()
```

Comment:
This test checks the current behavior and does not test a new feature at all. Right?

Reply:
Correct. Obviously we would not want multiple instances for a single schema, but already SchemaTypes only requests a single instance per type.